README.md

```diff
@@ -43,21 +43,21 @@ npm install
 
 ## New Token Methods
 
-Líf token is ERC20 compatible but it also has two more methods to allow the transference of data between users/contracts.
+Líf token is ERC20 compatible but it also has two more methods to allow the transference of data and execution of calls between users/contracts.
 
 ### transferData
 
 Transfer tokens from one address to another.
```

Comment:
The method name is `transferData` but it says it's used to transfer tokens. Is that correct? Is it because it actually transfer both tokens and data?